src/log.js

```diff
@@ -485,22 +485,19 @@ function pushIfNonEmpty(array, val) {
  * @return {!Error};
  */
 export function duplicateErrorIfNecessary(error) {
```

Comment:
@jridgewell Looks like you wrote this. I re-implemented it in what I thought was a more elegant way. WDYT?

Reply:
@jridgewell Let me know if this implementation works.